dependencies.props

```diff
@@ -6,7 +6,7 @@
   -->
   <PropertyGroup>
     <ProdConCurrentRef>7daa2e55de668c4c8ee832059aab04917153881e</ProdConCurrentRef>
-    <CoreClrCurrentRef>6a23464cf760231c209ca5c757aebe3c8bc26896</CoreClrCurrentRef>
+    <CoreClrCurrentRef>73c72e59ee47f3a6b8eaeb87507314209b17e128</CoreClrCurrentRef>
```

Comment:
@dagoo Why is there a coreclr ref but not others? Do we need an explicit coreclr ref or should it just be coming through prodcon?

Reply:
This is for the ILAsm tooling package version, not a submodule hash, so it has to be set up like the BuildTools updater. Unfortunately, right now there's only a package version updater for `txt` build-infos and only a submodule updater for build manifest `xml`s. In theory if we had both types of updater for build manifests we could do the ILAsm and submodule updates with the same `CurrentRef`.

Reply:
Ok. That is kind of annoying but I at least understand it now.